documentation/library_guide/extension_api.rst

```diff
@@ -313,3 +313,67 @@ Example of Range-based API usage
 
         copy(oneapi::dpl::execution::dpcpp_default, view, range_res);
     }
+
+Async API
+--------------------------
+
+Functions defined in the STL ``<algorithm>`` or ``<numeric>`` headers are traditionally blocking. oneDPL extends the functionality of C++17 parallel algorithms by providing asynchronous algorithm with non-blocking behavior. This experimental feature enables expressing concurrent control flow by building dependency chains and interleaving algorithm calls, as well as interoperability with SYCL kernels. 
+
+The current implementation for async algorithms is limited to DPC++ Execution Policies.
+All functionality described below is available in ``oneapi::dpl::experimental`` namespace.
```

Comment:
Please add the bold below: 

All **the** functionality described below is available in **the** ``oneapi::dpl::experimental`` namespace.